src/Forms/ConfirmedPasswordField.php

```diff
@@ -562,9 +598,18 @@ public function saveInto(DataObjectInterface $record)
             return;
         }
 
-        if (!($this->canBeEmpty && !$this->value)) {
-            parent::saveInto($record);
+        // Create a random password for Member if password is blank
```

Comment:
```suggestion
        // Create a random password if password is blank and the flag is set
```
Nothing to do with "member" as far as the field is concerned.

Reply:
Updated